src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -213,6 +152,54 @@ void CascadiaSettings::_CreateDefaultProfiles()
     _profiles.emplace_back(powershellProfile);
     _profiles.emplace_back(cmdProfile);
     
+    Profile WSLDistro{};
+    HANDLE readPipe;
+    HANDLE writePipe;
+    SECURITY_ATTRIBUTES sa{ sizeof(sa), nullptr, true };
+    if (CreatePipe(&readPipe, &writePipe, &sa, 0)) {
+        STARTUPINFO si{};
+        si.cb = sizeof(si);
+        si.dwFlags = STARTF_USESTDHANDLES;
+        si.hStdOutput = writePipe;
+        si.hStdError = writePipe;
+
+        PROCESS_INFORMATION pi{};
+        std::wstring command = L"wsl.exe --list";
+        HRESULT hr = CreateProcessW(nullptr, const_cast<LPWSTR>(command.c_str()), nullptr, nullptr,

```

Comment:
CreateProcess returns a bool, not an HRESULT.
https://msdn.microsoft.com/en-us/windows/desktop/ms682425